linked-list/linked-list.spec.js

```diff
@@ -0,0 +1,72 @@
+import LinkedList from './linked-list';
+
+describe('LinkedList', () => {
+  it('push/pop', () => {
```

Comment:
Could a better description be: `add/extract elements to the end of the list with push/pop`?